lua/wikis/rainbowsix/Infobox/Character/Custom.lua

```diff
@@ -0,0 +1,482 @@
+		IconImageWidget{
+			imageLight = 'R6S Para Bellum atk logo.png',
+			link = '',
+			size = '14px'
+		},
+		' Attack'
+	}
+}
+local TEAM_DEFENSE = HtmlWidgets.Fragment{
+	children = {
+		IconImageWidget{
+			imageLight = 'R6S Para Bellum def logo.png',
+			link = '',
+			size = '14px'
+		},
+		' Defense'
+	}
+}
+
+---@param label string
+---@return Widget
+local function createPriceLabel(label)
+	return HtmlWidgets.Sup{
+		children = {
+			HtmlWidgets.Span{
+				classes = { 'elm-bg' },
+				css = {
+					['font-family'] = 'monospace',
+					['font-weight'] = 'bold',
+					padding = '0 1px'
+				},
+				children = { label }
+			}
+		}
+	}
+end
+local STANDARD_EDITION_LABEL = createPriceLabel('SE')
+local SEASON_PASS_LABEL = createPriceLabel('SP')
+
+local OPERATOR_PRICES = {
+	launch = {
+		renown = 0,
+		credit = 300
+	},
+	['36m'] = {
+		renown = 10000,
+		credit = 240
+	},
+	['24m'] = {
+		renown = 15000,
+		credit = 360
+	},
+	['12m'] = {
+		renown = 20000,
+		credit = 480
+	},
+	['0m'] = {
+		renown = 25000,
+		credit = 600
+	}
+}
+OPERATOR_PRICES['36 months'] = OPERATOR_PRICES['36m']
+OPERATOR_PRICES['24 months'] = OPERATOR_PRICES['24m']
+OPERATOR_PRICES['12 months'] = OPERATOR_PRICES['12m']
+OPERATOR_PRICES['0 months'] = OPERATOR_PRICES['0m']
+
+---@class RainbowsixHeroInfobox: CharacterInfobox
+local CustomCharacter = Class.new(Character)
+local CustomInjector = Class.new(Injector)
+
+---@param frame Frame
+---@return Html
+function CustomCharacter.run(frame)
+	local character = CustomCharacter(frame)
+	character:setWidgetInjector(CustomInjector(character))
+	character.args.informationType = 'Operator'
+	return character:createInfobox()
+end
+
+---@param id string
+---@param widgets Widget[]
+---@return Widget[]
+function CustomInjector:parse(id, widgets)
+	local args = self.caller.args
+	if id == 'country' then
+		local ageCalculationSuccess, age = pcall(AgeCalculation.run, {
+			birthdate = args.birthdate,
+			birthlocation = args.birthplace,
+		})
+
+		return ageCalculationSuccess and {
+			Cell{name = 'Born', content = {age.birth}},
+		} or {}
+	elseif id == 'role' then
+		return WidgetUtil.collect(
+			Cell{
+				name = 'Team',
+				children = self.caller:_getTeam(args)
+			},
+			Cell{
+				name = 'Operator Role',
+				content = Array.map(
+					self.caller:getAllArgsForBase(args, 'function'),
+					function (role)
+						return Link{
+							link = ':Category:' .. role .. ' Operators',
+							children = role
+						}
+					end
+				),
+			}
+		)
+	elseif id == 'class' then
+		return {
+			Cell{
+				name = 'Team Rainbow',
+				content = self.caller:_getTeamRainbow(args),
+				options = { separator = ' ' }
+			},
+			Cell{
+				name = 'Affiliation',
+				content = { args.affiliation }
+			}
+		}
+	elseif id == 'release' then
+		if Logic.isEmpty(args.releasedate) then
+			return {}
+		end
+		local patchData = mw.ext.LiquipediaDB.lpdb('datapoint', {
+			conditions = '[[type::patch]] AND [[date::'.. args.releasedate ..']]',
+		})[1]
```

Comment:
if we are only interested in 1 element imho add `limit = 1` so the lpdb call doesn't try to fetch more (useless) entries after finding one